include/render.hpp

```diff
@@ -146,15 +146,21 @@ void render(sycl::queue& queue, std::array<color, width * height>& fb,
       sycl::buffer<color, 2>(fb.data(), sycl::range<2>(height, width));
   auto hittables_buf = sycl::buffer<hittable_t, 1>(hittables.data(),
                                                    sycl::range<1>(nb_hittable));
+  auto cam_buf = sycl::buffer<camera, 1>(&cam, sycl::range<1>(1));
```

Comment:
No need to have a `buffer` if the `cam` is read-only.
I guess the renderer does not update the camera?

Reply:
This commit was used for performance tests and trying to make the code similar to [code play sycl ray tracing example](https://www.codeplay.com/portal/blogs/2020/06/19/ray-tracing-in-a-weekend-with-sycl-part-2-pixel-sampling-and-material-tracing.html)

Its changeset has been discarded in the history.